test/index.js

```diff
@@ -88,7 +90,9 @@ test('JavaScript URL routing 2/2', t => {
 });
 */
 
-let rawdata = readFileSync('urlpatterntestdata.json');
+
+
```

Comment:
Why two newlines?

Reply:
this depends of https://github.com/kenchris/urlpattern-polyfill/pull/69, so it should be merged first!